frontend/js/src/explore/fresh-releases/components/ReleaseTimeline.tsx

```diff
@@ -6,10 +6,11 @@ import { formatReleaseDate, useMediaQuery } from "../utils";
 type ReleaseTimelineProps = {
   releases: Array<FreshReleaseItem>;
   order: string;
+  direction: string;
```

Comment:
Same comment as above:
```suggestion
  direction: SortDirection;
```